themes-default/slim/views/layouts/main.mako

```diff
@@ -145,6 +145,7 @@
                     routes
                 });
             }
+            Vue.use(window['vue-js-toggle-button'].default);
```

Comment:
@OmgImAlexis should I import here?

Reply:
If you ask me, the component is already imported on line 120, and I see no reason not to put this here.

Reply:
Yeah Vue is a global so you only need it once per page, if it's in the `main.mako` once it'll be on every single Vue instance on that page. No need to re-register it.

Reply:
@sharkykh if the imported script doesn't auto install then that line will still be needed.